bridge_adaptivity/bridge_lti/tests/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'Boiko'
```

Comment:
do we need this line? All authors already mentioned in the AUTHORS file?

Reply:
it was added by pycharm, not by me... I will delete it